src/os/CMakeLists.txt

```diff
@@ -115,16 +121,22 @@ endif()
 
 if(WITH_SPDK)
   include_directories("${CMAKE_SOURCE_DIR}/src/spdk/include")
+  add_custom_target(build_dpdk
+    COMMAND
+    make config T=x86_64-native-linuxapp-gcc && make EXTRA_CFLAGS="-fPIC"
+    WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}/src/spdk/dpdk
+    COMMENT "dpdk building")
```

Comment:
@liupan1111 in other words, we should be prepared for that. please note that in addition to amd64+linux+gcc, Ceph supports other combinations of arch, platforms and toolset. so we cannot assume the target is always "x86_64-native-linuxapp-gcc" here.